src/executables/util_cs.c

```diff
@@ -3286,6 +3314,10 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
 				     &applied_final_lsa);
 	  (void) db_shutdown ();
+	  if (error != NO_ERROR)
+	    {
+	      goto print_applyinfo_usage;
+	    }
```

Comment:
```suggestion

```
I think this code is not necessary. 
error check will be done after this block.

Reply:
Same comment exists above, so I copied the answer.

In this area, there is no need to be printing error message.
[applyinfo -p](https://www.cubrid.org/manual/ko/11.0/ha.html?highlight=applyinfo#cmdoption-applyinfo-p) as we referenced above link, this statement is designed for error flow by -p option.

Previous code is also verified in the -p option error in the la_log_page_check(). So I decided to just add the flow when the errors arise.

if the below exists, applyinfo utility emits nothing from this point in the case of -p option error.
if not, print "Delay in Copying Active Log"